schemas/shacl/valid.jsonld

```diff
@@ -0,0 +1,44 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
+    "codemeta": "https://codemeta.github.io/terms/",
+    "repostatus": "https://www.repostatus.org/#"
+  },
+  "@type": "SoftwareSourceCode",
+  "codeRepository": {
+    "@id": "https://github.com/netwerk-digitaal-erfgoed/network-of-terms"
+  },
+  "contIntegration": "https://github.com/netwerk-digitaal-erfgoed/network-of-terms/actions",
+  "dateCreated": "2020-04-17",
+  "datePublished": "2020-09-24",
+  "issueTracker": "https://github.com/netwerk-digitaal-erfgoed/network-of-terms/issues",
+  "name": "Network of Terms",
+  "description": "GraphQL API for searching the Network of Terms",
+  "license": {
+    "@id": "https://spdx.org/licenses/EUPL-1.2"
```

Comment:
@proycon Not sure if this should include `.html`: https://spdx.org/licenses/EUPL-1.2.html

Reply:
No, I've seen references without .html in multiple places, including their own documentation and that also makes more sense I think. Ideally they should implement proper content negotation so that these URLs might also serve jsonld/turtle/etc and not just html.